geth/jail/jail_test.go

```diff
@@ -115,6 +116,44 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+	var wg sync.WaitGroup
```

Comment:
Yes but `wg` is not part of test setup but a thing which controls test flow during Run.